rest/taskana-rest-spring-example-wildfly/src/test/java/pro/taskana/example/wildfly/AbstractAccTest.java

```diff
@@ -78,4 +85,52 @@ protected String parseServerLog() throws Exception {
     }
     return stringBuilder.toString();
   }
+
+  private static void stopPostgresDb() {
+    try {
+      boolean isWindows = System.getProperty("os.name").toLowerCase().startsWith("windows");
+      ProcessBuilder builder = new ProcessBuilder();
+      if (isWindows) {
+        builder.command(
+            "cmd.exe", "/c", "docker-compose -f ../../docker-databases/docker-compose.yml down");
```

Comment:
Do we need the "-v" only for start, or do we need it here too? ("... down -v")

Reply:
good catch. Was a mistake. We only need `down -v`.